packages/core/core/src/requests/TargetRequest.js

```diff
@@ -154,7 +154,7 @@ export class TargetResolver {
       optionTargets = [exclusiveTarget];
     }
 
-    let packageTargets = await this.resolvePackageTargets(
+    let packageTargets: Map<string, ?Target> = await this.resolvePackageTargets(
```

Comment:
nit: `Map<string, Target | null>`? It's not really a maybe target, it's more a target or the "skipped" value null